deepspeed/runtime/config.py

```diff
@@ -905,6 +909,10 @@ def _initialize_params(self, param_dict):
             USE_NODE_LOCAL_STORAGE_CHECKPOINT,
             USE_NODE_LOCAL_STORAGE_CHECKPOINT_DEFAULT)
 
+        data_types_params = get_data_types_params(param_dict)
```

Comment:
@mrwyattii to confirm if we have pydantic support for the main runtime config yet?

Reply:
We do not for the main runtime config. That will be a larger refactor, but I would like to prioritize it and get all the configs using pydantic.

Reply:
No problem, just wanted to make sure we didn't add a new config using the old approach if it was already converted. I imagine the main runtime config will be a bigger undertaking than the others.